src/message/action.js

```diff
@@ -917,8 +919,22 @@ function Originate() {
  * @property {String} StateInterface Optional, State interface
  * @augments Action
  */
-function QueueAdd() {
+function QueueAdd(queue, interface, memberName, penalty, stateInterface) {
```

Comment:
Let's leave optional arguments out of the function variables, so we can keep the standard nami behavior to add options, like in [this example](https://github.com/marcelog/Nami/blob/master/src/index.js#L73), one only has to instantiate the action and just set whatever fields are needed without having to have a lot of arguments when instantiating the action itself (this allow us to have a cleaner interface)